modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -50,24 +51,54 @@ const executeKeydownOverride = (editor: Editor, caret: Cell<Text | null>, evt: K
   });
 };
 
-const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent) => {
+const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent, isBackspaceKeydown: boolean) => {
+  const platform = PlatformDetection.detect();
+  const os = platform.os;
+  const browser = platform.browser;
+  const multiDeleteKeyPatterns: MatchKeys.KeyPattern[] = os.isMacOS() ? [
+    { keyCode: VK.BACKSPACE, altKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+    { keyCode: VK.DELETE, altKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+  ] : [
+    { keyCode: VK.BACKSPACE, ctrlKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+    { keyCode: VK.DELETE, ctrlKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) }
+  ];
+
+  // macOS surpresses keyup events for most keys including Backspace when Meta key is engaged
+  // To emulate Meta + Backspace on macOS, add a pattern for the meta key when backspace was
+  // detected on keydown
+  if (os.isMacOS() && isBackspaceKeydown) {
+    multiDeleteKeyPatterns.push({
+      // firefox detects Cmd as "Command" not "Meta"
+      keyCode: browser.isFirefox() ? 224 : 91,
+      action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor)
+    });
+  }
+
   MatchKeys.execute([
     { keyCode: VK.BACKSPACE, action: MatchKeys.action(CefDelete.paddEmptyElement, editor) },
-    { keyCode: VK.DELETE, action: MatchKeys.action(CefDelete.paddEmptyElement, editor) }
+    { keyCode: VK.DELETE, action: MatchKeys.action(CefDelete.paddEmptyElement, editor) },
+    ...multiDeleteKeyPatterns
   ], evt);
 };
 
 const setup = (editor: Editor, caret: Cell<Text | null>): void => {
+  // global backspace keydown state to emulate Meta + Backspace keyup detection on macOS
```

Comment:
This isn't really a global state that would be for the hole page in my mind.

Reply:
True, I clarified the comment to
```
// track backspace keydown state for emulating Meta + Backspace keyup detection on macOS
let isBackspaceKeydown = false;
```